libs/editor/build.gradle

```diff
@@ -34,6 +34,13 @@ android {
         targetSdkVersion rootProject.targetSdkVersion
     }
 
+    buildTypes {
+        debug {
+            minifyEnabled true
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard.cfg'
```

Comment:
Just and FYI and as per [the documentation](https://developer.android.com/build/shrink-code#enable) when `proguardFiles` is configured then this includes the default ProGuard rules files that are packaged with the Android Gradle plugin.

PS: I am not sure this will work as there is no `proguard.cfg` file for the `editor` module, we only got that on the `WordPress` module. 🤔